site/src/components/PaginationWidget/PaginationWidgetBase.tsx

```diff
@@ -38,59 +39,88 @@ export const PaginationWidgetBase = ({
         display: "flex",
         flexDirection: "row",
         padding: "20px",
+        columnGap: "6px",
       }}
     >
-      <Button
-        css={{
-          marginRight: 4,
-        }}
+      <PaginationNavButton
+        disabledMessage="You are already on the first page"
+        disabled={onFirstPage}
         aria-label="Previous page"
-        disabled={isFirstPage}
         onClick={() => {
-          if (!isFirstPage) {
-            onChange(page - 1);
+          if (!onFirstPage) {
+            onPageChange(currentPage - 1);
```

Comment:
could we rename this to `onChangePage`? just feels a lot more natural to me

Reply:
Yeah, I generally go with the `onNoun` structure. You can have a page change, but you can't have a change page, if that makes sense?

Reply:
I usually think of it as `onAction` 🤔 "a change" is kind of a weird "noun". I won't block on it, just my 2¢

Reply:
Just double-checked MDN, and all the more specialized change-based follow the `onXChange` convention
https://developer.mozilla.org/en-US/docs/Web/Events

I am open to changing prop names for clarity, but my thinking is that it's best to mirror the base HTML standards as closely as possible (same reason why I agree with extending the base `HTMLAttributes` when possible)